haxe.html.markdown

```diff
@@ -495,8 +495,10 @@ class LearnHaxe3{
         // foo_instance.public_read = 4; // this will throw an error if uncommented:
         // trace(foo_instance.public_write); // as will this.
 
-        trace(foo_instance + " is the value for foo_instance"); // calls the toString method
-        trace(foo_instance.toString() + " is the value for foo_instance.toString()"); // same thing
+         // calls the toString method
```

Comment:
OK. I actually considered adding a colon, but decided not to do so in order to maintain the same style as the rest of the code, which is mainly without colons.
I hope I'll figure out how to make this changes, as I'm new here...